staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -67,6 +67,22 @@ const timeFmt = "2006-01-02T15:04:05.999"
 // undesired becomes completely unused, all the config objects are
 // read and processed as a whole.
 
+const (
+	// Borrowing among priority levels will be accomplished by periodically
+	// adjusting the current concurrency limits (CurrentCLs);
+	// borrowingAdjustmentPeriod is that period.
+	borrowingAdjustmentPeriod = 10 * time.Second //nolint:golint,unused
+
+	// The input to the borrowing is smoothed seat demand figures.
+	// Every adjustment period, each priority level's new smoothed demand is set to
+	// seatDemandSmoothingCoefficient times the previous smoothed demand
+	// plus (1-seatDemandSmoothingCoefficient) times the recent envelope
+	// of seat demand.  The particular number appearing here has the property
+	// that half of the current smoothed value comes from the smoothed value
+	// of 5 minutes ago.
```

Comment:
This is probably unnecessarily paranoid, but can you document that we're allowed to change this in the future if we find a better decay length, we're not promising this will never change?

Reply:
sure